api/src/accounts.rs

```diff
@@ -193,25 +208,67 @@ pub struct Account {
 }
 
 impl Account {
-    /// Creates a new account struct and determines the current ledger info, and determines the
-    /// ledger version to query
     pub fn new(
         context: Arc<Context>,
         address: Address,
         requested_ledger_version: Option<U64>,
         start: Option<StateKey>,
         limit: Option<u16>,
+        no_internal_indexer_required: bool,
```

Comment:
Can we flip it? i.e. "require_state_indices: bool"

Reply:
how about other APIs, can we resort to the list of all APIs that possibly rely on the internal indexer and make sure a similar treatment is done for them?

Reply:
fliped.  if you mean, have I checked all APIs to find the one relying on internal indexer, yes.